wc-admin.php

```diff
@@ -47,6 +47,14 @@ function dependencies_satisfied() {
 			&& class_exists( 'WooCommerce' ) && version_compare( WC_VERSION, '3.5', '>' );
 }
 
+/**
+ * Daily events to run.
+ */
+function wc_admin_daily() {
```

Comment:
Not sure if we have naming conventions in place for this, but could we make this different from the event name?  (e.g., `do_wc_admin_daily()` so we can distinguish a bit faster).

Reply:
Can do